server/src/test/java/org/elasticsearch/search/aggregations/bucket/DateRangeIT.java

```diff
@@ -297,9 +299,11 @@ public void testSingleValueFieldWithStringDatesWithCustomFormat() throws Excepti
     }
 
     public void testSingleValueFieldWithDateMath() throws Exception {
-        ZoneId timezone = randomZone();
+        ZoneId timezone = ZoneOffset.UTC;
```

Comment:
leftover and should be a random zone again?

Reply:
yes, should be random again

Reply:
oh sure, thanks for clarifying...